### DISCUSSION TITLE
How to correctly import my modules?

### DISCUSSION THREAD

**Discussion by semcha** [PrefectHQ/prefect#4330] [Q&A] [answered]
Hi! How I can correctly import my own module (.py file)?
For example:

dwh (folder)
---db_utlis.py
---load_api_task.py (code with Flow)

If I type `import db_utils` in load_api_task.py file I get an error `ModuleNotFoundError: No module named 'db_utils'` when I trying register flow by `prefect register --project myproject --path /home/admin/dwh/` command.

If I call export `PYTHONPATH=/home/admin/dwh/` before prefect register then flow registration is successful. But when I start this flow I got an error: `Failed to load and execute Flow's environment: ModuleNotFoundError("No module named 'db_utils'")`

What the correct way to import my module in this example?

**Comment by zanieb** [comment, marked as answer]
Hi! The registration half of this is fixed in https://github.com/PrefectHQ/prefect/pull/4332 but you should note that you'll need to provide the `PYTHONPATH` to your agent as well for the flow to know where to look for dependencies at runtime.

**Reply by semcha** [reply]
Hi @madkinsz! Do you mean `--import-path` option in `prefect agent local start`?

**Reply by zanieb** [reply]
Well, the way that's implement looks like this
```python
        if os.environ.get("PYTHONPATH"):
            python_path.append(os.environ["PYTHONPATH"])
        if self.import_paths:
            python_path.extend(self.import_paths)
```
so you can provide it as an environment variable or by `--import-path`

**Reply by zanieb** [reply]
More broadly speaking, I'd recommend you have `setup.py` and install your python module in a docker image and use a `DockerRun` so this scales beyond a `LocalAgent`

**Reply by semcha** [reply]
@madkinsz Thank you very much!
